kafka/src/main/java/org/apache/apex/malhar/kafka/OneToManyPartitioner.java

```diff
@@ -52,8 +52,8 @@ public OneToManyPartitioner(String[] clusters, String[] topics, AbstractKafkaInp
       for (Map.Entry<String, List<PartitionInfo>> topicPartition : clusterMap.getValue().entrySet()) {
         for (PartitionInfo pif : topicPartition.getValue()) {
           int index = i++ % partitionCount;
-          if (eachPartitionAssignment.get(index) == null) {
```

Comment:
This will throw ArrayIndexOutOfBoundaryException  even the space is pre-allocated but not assigned